### DISCUSSION TITLE
SDXL Lightning 1 Step | 2 Step | 4 Step | 8 Step

### DISCUSSION THREAD

**Discussion by kakachiex2** [comfyanonymous/ComfyUI#2867] [General]
Hello everyone, I post this about SDXL Lightning here you can find Models and workflow for ComfyUI ::
https://huggingface.co/ByteDance/SDXL-Lightning
I upgrade the original workflow to support multi lora and multi model merge and is really fast.
![Screenshot 2024-02-22 013549](https://github.com/comfyanonymous/ComfyUI/assets/1762112/ae3cbd78-b282-43cc-b906-ea8f3b4ca0bf)
![image (53)](https://github.com/comfyanonymous/ComfyUI/assets/1762112/3e4942e9-ba75-4a79-b6d3-7d5446923675)
![image (50)](https://github.com/comfyanonymous/ComfyUI/assets/1762112/297ea4a9-90a0-4b74-9541-714c77e5f2d3)
![ComfyUI_00030_](https://github.com/comfyanonymous/ComfyUI/assets/1762112/dcb5ab0f-f104-4261-8a39-a5d18b5a30f7)

Here is my Workflow base on lora not the full checkpoint.
![SDXL_Lighning_Lora_WF_V1 0](https://github.com/comfyanonymous/ComfyUI/assets/1762112/e45c2d4d-f3bd-4999-aa53-b7ffcd4f54db)

**Comment by Piscabo** [comment]
Thanks for that. I suppose the models you merge have a lot to do with the quality. I can't get such clear images. Is there another trick?

**Reply by kakachiex2** [reply]
Use my workflow and it have the model listed you only will have to download it, the workflow is embed in the last image, Im using 8 steps but 4 and two give good results

**Comment by kakachiex2** [comment]
This is Lightning lora and LCM lora combined and looks the quality in 2 steps
![ComfyUI_temp_tjltv_00012_](https://github.com/comfyanonymous/ComfyUI/assets/1762112/2ee16ffc-722e-453c-a35e-34231fb89adb)

**Comment by JorgeR81** [comment]
Beautiful images ! Thanks for sharing. 

By the way, where did you get that Comfy UI theme ?
Is that from here:
https://github.com/yolain/ComfyUI-Easy-Use

**Reply by kakachiex2** [reply]
Yes this is one of them

**Reply by kakachiex2** [reply]
Tiny Terra node give some UI relate change too.

**Comment by dicksondickson** [comment]
This is awesome work! I want to try out your workflow but I'm having trouble finding this missing node "Diffusers_X_Adapter". Can you tell me which custom nodes you installed?

**Comment by kakachiex2** [comment]
I'm not using diffuser X-Aapter in this workflow this is estrange download the workflow image and load it.
Node used are:
- Node everywhere
- CR Custom node and ComfyUI native Vae, Ksampler and...

**Comment by kakachiex2** [comment]
I recommend using the lora for acceleration not the model, the lora give much better result combined with other model with the merge node.
